src/arrayMethodSort.js

```diff
@@ -4,9 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
```

Comment:
The default compare function should return a number rather than a boolean. It should return a negative value if 'a' is less than 'b', zero if they're equal, and a positive value if 'a' is greater than 'b'.